tools/test-case-generators/generate-test-cases

```diff
@@ -85,6 +85,12 @@ def main(distro, arch, image_types, keep_image_info, store, output):
         test_case_request["compose-request"]["distro"] = distro
         test_case_request["compose-request"]["arch"] = arch
         test_case_request["compose-request"]["repositories"] = repos_dict[distro][arch]
+
+        # overwrite the compose request with any distro-specific compose request options
+        if distro in test_case_request:
+            test_case_request["compose-request"].update(test_case_request[distro])
+            del test_case_request[distro]
```

Comment:
Feels a bit iffy to have to delete the entry, could you mabe just create a new test_case_request and insert the things you want, so there is no need to delete anyhing from it?

Reply:
sure!